src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+
+const array = [
+  {
+    description: 'Shopping',
+    completed: false,
+  },
+  {
+    description: 'Going to school',
+    completed: false,
+  },
+  {
+    description: 'Washing the dishes',
+    completed: false,
+  },
+];
```

Comment:
- [ ] Your tasks should have `description`, `completed` and `index` properties. Kindly add the `index` property